### DISCUSSION TITLE
如何解析局域网内，指定机器的ipv6地址？

### DISCUSSION THREAD

**Discussion by Jengro777** [jeessy2/ddns-go#800] [Q&A]
局域网内有多台机器，我只想在一台控制机上面部署ddns-go。

我是否可以通过内网ip 或者 mac地址 的方式，解析非控制机的 ipv6地址 呢？

**Comment by Jengro777** [comment]
第二个问题是：

如果我的主控机是widows ，其它机器是linux  ，又是否可以通过内网静态ip 或者 mac地址的方式，解析其它linux机器上面的 ipv6 地址呢？

**Comment by WaterLemons2k** [comment]
可以参考 https://github.com/jeessy2/ddns-go/issues/795#issuecomment-1658438271 和 https://github.com/jeessy2/ddns-go/issues/795#issuecomment-1658609102 获取其它 Linux 机器的 IPv6 地址。
